ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/DB/Util/AcidState.hs

```diff
@@ -101,3 +107,56 @@ zoomAll l upd = StateT $ \large -> do
 mustBeRight :: Either Void b -> b
 mustBeRight (Left  a) = absurd a
 mustBeRight (Right b) = b
+
+type MonadAcidCleanup m =
```

Comment:
It's used in exactly one place, which means that it's useless (use these constraints directly).